### ISSUE TITLE
fresh cosmo_* tests failures under wheezy/sparc

### ISSUE THREAD

**Issue by yarikoptic** [PyMVPA/PyMVPA#229]
Please check it out at
http://nipy.bic.berkeley.edu/builders/pymvpa-py2.7-wheezy-sparc/builds/14/steps/shell_6/logs/stdio
If those are sparc specific (ie do not reproduce in neurodebian wheezy vm) -- I will give you access to that beast

**Comment by nno** [comment]
This test now fails on my machine (OSX 10.9.5) with numpy 1.9.0 and scipy 0.14.0.
On NeuroDebian it runs fine with numpy 1.6.2 and scipy 0.10.0.
I'll try to figure out which of these components is causing it and try to fix it.

**Comment by nno** [comment]
On NeuroDebian with the latest numpy (1.10.0.dev0+63e8bab) and scipy (0.16.0.dev0+1306448) the tests fail as well. The following code pins down the issue:
```
import numpy as np
arr=np.array

x=arr([[1]])
y=arr([[1,3]])

xy=arr([[x,y]], dtype='O')
```

which crashes with recent numpy and scipy:
```
Traceback (most recent call last):
  File "test.py", line 7, in <module>
    xy=arr([[x,y]], dtype='O')
ValueError: could not broadcast input array from shape (2) into shape (1)
```

**Comment by nno** [comment]
This issue should be addressed by PR #299.

**Comment by yarikoptic** [comment]
#299 was merged, thus closing
